inference-engine/src/vpu/graph_transformer/src/middleend/pass_manager.cpp

```diff
@@ -277,8 +280,8 @@ PassSet::Ptr PassManager::buildMiddleEnd() {
     // Model SW-specific optimizations after data layout adjustment
     //
 
-    ADD_PASS(mergeEltwiseAndReLU);
-    ADD_DUMP_PASS("mergeEltwiseAndReLU");
+    ADD_PASS(mergeEltwiseAndReLUStatic);
```

Comment:
Is it also possible to move the static merge right before the data layout pass?

Reply:
No, static version checks strides that're aligned in adjustDataLayout

Reply:
I just wonder what happened if the `Eltwise+ReLU` pair is static in dynamic network, why the processing is different for static `Eltwise+ReLU` pair in dynamic network and in fully static network?

Reply:
It's supposed to be explained in PR description and commit message. Is it unclear to you after reading it?
Do you mean it's better to add comment in code?

Reply:
I mean the reordering `mergeEltwiseAndReLU` and `adjustDataLayout`, you only say that that it gives performance gain for dynamic cases, but it was unclear for me what is happening with static cases in dynamic networks. So do I correctly understand that once the performance degradation will be investigated, we will be able both to enable merging for fully static network as well as reordering `mergeEltwiseAndReLUStatic` before `adjustDataLayout` for all cases?

Reply:
> what is happening with static cases in dynamic networks

There is no such a thing as *static in dynamic* or *dynamic in static*. Network is dynamic or static, there is no 3rd option. If it is dynamic we do Eltwise + ReLU merge as expected, otherwise merge logic is unchanged.

> So do I correctly understand that once the performance degradation will be investigated, we will be able both to enable merging for fully static network as well as putting mergeEltwiseAndReLUStatic before adjustDataLayout?

I'm not sure I got *we will be able both to enable merging* part right, but once degradation root cause is fixed it'd be possible to do Eltwise + ReLU merge as expected at all cases.

Reply:
Ok, got it.